WooCommerce/Classes/ViewRelated/Reviews/ReviewsViewModel.swift

```diff
@@ -93,9 +130,9 @@ final class ReviewsViewModel {
 extension ReviewsViewModel {
     /// Prepares data necessary to render the reviews tab.
     ///
-    func synchronizeReviews(pageNumber: Int = Settings.firstPage,
-                            pageSize: Int = Settings.pageSize,
-                            onCompletion: (() -> Void)? = nil) {
+    func synchronizeReviews(pageNumber: Int,
+                            pageSize: Int,
+                            onCompletion: (() -> Void)?) {
```

Comment:
Could you clarify the reason why we need to remove the default values here?

Reply:
1. I introduced a protocol that defines this method and we cannot have methods with default parameters in protocols.
1. There was no need for the default parameter at all, except in the `ReviewViewModelTests`. 

So, I decided to remove those and pass in default values from tests, just like what other test classes follow. Kindly let me know if you think that this isn't alright.